src/app/testing/page.tsx

```
@@ -1,3 +1,15 @@
-export default function Testing({}) {
+"use server";
 
+import { getSBBrowserClient } from "@/lib/supabase/sbClient";
+
+export default async function Testing({}) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)